pvnet/models/multimodal/encoders/encoders3d.py

```diff
@@ -201,6 +202,84 @@ def forward(self, x):
         return out
 
 
+class ResConv3DNet2(AbstractNWPSatelliteEncoder):
```

Comment:
Same here, could we use a different name? Is there some architectural thing that we could include in the name so it's not something2?

Reply:
The reason for this was that the structure of this is the same as the `ResFCNet2` network. And `ResFCNet2` and `ResFCNet` differed by activation functions and dropout I think. So yeh we could combine them. I'd rather do that in a second pull request than tag it on to this though